packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.tsx

```diff
@@ -54,6 +58,19 @@ export interface PresentationInstanceFilterDialogProps {
   initialFilter?: PresentationInstanceFilterInfo;
 }
 
+/** Set of custom handlers that will additionally be called on each action respectively. */
+export interface ToolbarHandlers {
+  onApplyHandler: () => void;
+  onCloseHandler: () => void;
+  onResetHandler: () => void;
+}
+
+/** Props required for rendering a toolbar */
+export interface ToolbarRendererProps {
+  toolbarHandlers: ToolbarHandlers;
+  getFilterInfo: (options?: BuildFilterOptions | undefined) => PresentationInstanceFilterInfo | undefined;
+}
```

Comment:
These are used in `@beta` APIs, so should also be `@beta`. Also, since they're tightly bound to the filtering dialog, the name should be changed to tell that.